lib/converters/converter-v2-to-v1.js

```diff
@@ -37,6 +37,28 @@ var _ = require('lodash').noConflict(),
 _.extend(Builders.prototype, {
 
     /**
+     * Extracts all subfolders from a v2 collection or folder
+     *
+     * @param folderOrCollection
+     */
+    extractFolderItems: function (folderOrCollection) {
+        var i,
+            self = this,
+            folders = [],
+            items = folderOrCollection.item || folderOrCollection.items;
+
+        !_.isArray(items) && (items = [items]);
```

Comment:
Optionally, `_.castArray`(https://lodash.com/docs/4.17.4#castArray) could be used here as well.